manager/src/test/resources/application.properties

```diff
@@ -17,4 +17,7 @@ event-bridge.kafka.security.protocol=PLAINTEXT
 
 # RHOAS Client
 event-bridge.rhoas.instance-api.host=http://test-instance-api.redhat.com
-event-bridge.rhoas.mgmt-api.host=http://test-management-api.redhat.com
\ No newline at end of file
+event-bridge.rhoas.mgmt-api.host=http://test-management-api.redhat.com
+
+# Schedule WorkManager every 1s for tests
+event-bridge.resources.workers.schedule=* * * * * ?
```

Comment:
I think it would be better to add `quarkus.scheduler.enabled=false` in the tests

Reply:
Agreed. Done in a new commit.

Reply:
> Do you think we are ok in terms of test coverage? I do not see any test on the workers.

@r00ta I will write some unit tests (in addition to their coverage by `ProcessorSeviceTest`).

Reply:
@r00ta More tests added.